Azure/sonic-buildimage
# [sonic-quagga]: Fix ipv6 nexthop ifindex.

**- Why I did it**
sonic-quagga repository has new fix. Update submodule to bring the fix into the image.

**- How I did it**
git submodule update --remote ...

**- How to verify it**
Build an image. Run on your dut. Check the situation when:
1. You have lo interfaces with address x.x.x.x
2. You have a dynamic session which uses x.x.x.x as a source address
3. You have a ipv6 prefix updates through this session to the device.
4. Check that the sessions nexthop interface is not equal to 'lo' interface, and point to correct nexthop interface.
**- Description for the changelog**



**- A picture of a cute animal (not mandatory but encouraged)**
